test/test_lab.py

```diff
@@ -0,0 +1,86 @@
+from cepimose.types import LabDashboard
+import unittest
+import cepimose
+import datetime
+from nose.plugins.attrib import attr
+
+# ? TODO some more assertions
+class CepimoseTestCase(unittest.TestCase):
+    def setUp(self):
+        super().setUp()
+
+    def tearDown(self):
+        super().tearDown()
+
+    def test_lab_start_timestamp(self):
+        ts = cepimose.lab_start_timestamp()
+        print("First NIJZ Covid-19 report:", ts)
+        first_case = datetime.datetime(2020, 3, 4)
+        today = datetime.datetime.now()
+
+        self.assertEqual(first_case, ts)
+
+    def test_lab_end_timestamp(self):
+        ts = cepimose.lab_end_timestamp()
+        print("Last update:", ts)
+        day_delta = datetime.timedelta(days=3)
+        today = datetime.datetime.now()
+        diff = today - ts
+        self.assertGreaterEqual(day_delta, diff)
+
+    def test_lab_PCR_tests_performed(self):
+        performed_PCR = cepimose.lab_PCR_tests_performed()
+        self.assertGreaterEqual(performed_PCR, 0)
+
+    def test_lab_PCR_total_tests_performed(self):
+        performed_PCR_total = cepimose.lab_PCR_total_tests_performed()
+        self.assertGreaterEqual(performed_PCR_total, 0)
```

Comment:
This is a cumulative sum of all times, so we can require a higher number, no?
```suggestion
        self.assertGreaterEqual(performed_PCR_total, 1300000)
```

Reply:
Similar in all `_total_` tests.

Reply:
Tnx for doing so in 55fa08e!